src/twisted/newsfragments/10045.feature

```diff
@@ -0,0 +1,2 @@
+Trial will now skip a test when the `skip` attribute is an empty string.
+In previous version the test was skipped only when the attribute was None.
```

Comment:
In previous versions the test was skipped when the attribute was a non-empty string or when it was None, I guess.  So maybe a good second sentence would be:

> In previous versions the test was not skipped in this case.